src/renderer/lib/DownloadsPanel.css

```
@@ -28,25 +28,97 @@
+    overflow: hidden;
+  }
+
+  /* Filled with currentColor, which is what lets the row's state colour the
+     bar without a rule per state. --kv-progress is the only inline style: it
+     is the datum, not the styling. */
+  .kv-downloads__bar::before {
+    content: "";
+    display: block;
+    width: var(--kv-progress, 0%);
+    height: 100%;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)